src/transformers/models/llama/tokenization_llama.py

```diff
@@ -255,7 +255,7 @@ def tokenize(self, text: "TextInput", add_special_tokens=False, **kwargs) -> Lis
         if self.add_prefix_space:
             text = SPIECE_UNDERLINE + text
 
-        tokens = super().tokenize(text, add_special_tokens=add_special_tokens, **kwargs)
+        tokens = super().tokenize(text, **kwargs)
```

Comment:
we should also remove it from the  ` def tokenize(self, text: "TextInput", add_special_tokens=False, **kwargs) -> List[str]:`. 
Also we need to update the `T5Tokenizer`